qa/TL0_python-self-test/test.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash -e
+set -o errexit
```

Comment:
Isn't `set -e` just a shortcut for `set -o errexit`?

From `help set`:
```
-o option-name
          Set the variable corresponding to option-name:
              errexit      same as -e
```

Reply:
I don't have particularly strong preference here, it just seemed a bit confusing. I guess doing consistently one way or the other has some value.